lstchain/spectra/data/sensitivity.txt

```diff
@@ -0,0 +1,16 @@
+#E_med [TeV] | % Crab
```

Comment:
Why is this data file needed?

Reply:
Ah sorry, just added it to compare in the plots the mc results with some preliminary LST sensitivity with real data. I can keep it out of the PR to avoid the dependency.